hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -322,17 +322,17 @@
   public static final int OZONE_SCM_PIPELINE_OWNER_CONTAINER_COUNT_DEFAULT = 3;
   // Pipeline placement policy:
   // the max number of pipelines can a single datanode be engaged in.
-  public static final String OZONE_DATANODE_MAX_PIPELINE_ENGAGEMENT =
-          "ozone.scm.datanode.max.pipeline.engagement";
+  public static final String OZONE_DATANODE_PIPELINE_LIMIT =
+          "ozone.datanode.pipeline.limit";
   // Setting to zero by default means this limit doesn't take effect.
-  public static final int OZONE_DATANODE_MAX_PIPELINE_ENGAGEMENT_DEFAULT = 2;
+  public static final int OZONE_DATANODE_PIPELINE_LIMIT_DEFAULT = 2;
 
   // Upper limit for how many pipelines can be created.
   // Only for test purpose now.
-  public static final String OZONE_SCM_PIPELINE_NUMBER_LIMIT =
+  public static final String OZONE_SCM_RATIS_PIPELINE_LIMIT =
       "ozone.scm.pipeline.number.limit";
```

Comment:
Can we change "ozone.scm.pipeline.number.limit" to "ozone.scm.pipeline.limit"?

Reply:
Yea I removed NUMBER but added RATIS. Thought it would show this config only applied for Ratis pipeline.

Reply:
How about ozone.scm.ratis.pipeline.limit?

Reply:
Sure. I've just updated the name.